CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+
+## Recent Changes
+
+- Simplify the preparation for JCL system tests
```

Comment:
not sure this requires a new patch to be published.
However we can combine the contents of this PR (after merged) with:
- #152